Dockerfile

```diff
@@ -12,4 +12,44 @@ FROM base as dev
 ENTRYPOINT poetry run flask run -h 0.0.0.0 -p 5000
 FROM base as prod
 ENV FLASK_ENV=production
-ENTRYPOINT poetry run gunicorn "app:create_app()" --bind 0.0.0.0:5000
\ No newline at end of file
+ENTRYPOINT poetry run gunicorn "app:create_app()" --bind 0.0.0.0:5000
+
+FROM base as development
+COPY ./pyproject.toml ./
+RUN poetry install
+ENTRYPOINT poetry run flask run --host=0.0.0.0
+
+FROM base as test
+COPY ./pyproject.toml .
+RUN poetry install
+
+
+# Install the latest versions of Mozilla Firefox and Geckodriver
+RUN export DEBIAN_FRONTEND=noninteractive && apt-get update \
+  && apt-get install --no-install-recommends --no-install-suggests --assume-yes \
+    curl \
+    bzip2 \
+    libgtk-3-0 \
+    libdbus-glib-1-2 \
+    xvfb \
+  && FIREFOX_DOWNLOAD_URL='https://download.mozilla.org/?product=firefox-latest-ssl&os=linux64' \
+  && curl -sL "$FIREFOX_DOWNLOAD_URL" | tar -xj -C /opt \
+  && ln -s /opt/firefox/firefox /usr/local/bin/ \
+  && BASE_URL='https://github.com/mozilla/geckodriver/releases/download' \
+  && VERSION=$(curl -sL 'https://api.github.com/repos/mozilla/geckodriver/releases/latest' | grep tag_name | cut -d '"' -f 4) \
```

Comment:
This line was failing for me, but I'm not sure if it could be a change from firefox - this does match [other examples](https://github.com/blueimp/geckodriver/blob/master/Dockerfile). A workaround I found was to adjust the `/latest` to be a specific release (e.g. `/36382341`) which isn't ideal but could suffice for now